packages/larva-patterns/modules/age-gate/age-gate.twig

```diff
@@ -0,0 +1,26 @@
+<div class="age-gate // {{ age_gate_classes }}">
+	<div class="age-gate-inner // {{ age_gate_inner_classes }}">
+		<div class="age-gate-header // {{ age_gate_header_classes }}">
+			{% include "@larva/components/c-svg/c-svg.twig" with c_svg %}
+			{% if c_heading %}
+				{% include "@larva/components/c-heading/c-heading.twig" with c_heading %}
+			{% endif %}
+		</div>
+		<div class="age-gate-form // {{ age_gate_form_classes }}">
+			<form>
+				{% if o_text_input_group %}
+					{% include "@larva/objects/o-text-input-group/o-text-input-group.twig" with o_text_input_group %}
+				{% endif %}
+				<div class="age-gate-error // {{ age_gate_error_classes }}"></div>
+				<div class="button-wrapper">
```

Comment:
Can this be prefixed to be `age-gate-button-wrapper`? The idea is to deter styling for any potentially generic selectors.

Reply:
Fixed! :)